src/bezpath.rs

```diff
@@ -96,8 +115,36 @@ impl VelloVector for VelloBezPath {
 
         path
     }
+
+    fn border_translation(&self, time: f64) -> DVec2 {
+        let (path, t) = self.inbetween(time);
+
+        let current = path.0.end_point().unwrap_or_default();
+        let point = interp_pathel(current, path.1, t as f32)
+            .end_point()
+            .unwrap()
+            .to_vec2();
+
+        DVec2::new(point.x, point.y)
+    }
+
+    fn border_rotation(&self, time: f64) -> f64 {
+        let (path, t) = self.inbetween(time);
+
+        let current = path.0.end_point().unwrap_or_default();
+        let before = interp_pathel(current, path.1, t as f32 - 1e-4)
```

Comment:
usually for like approximations and float testing this is common but it can lead to some issues. if, in some weird system, theres floating precision issues and this results in just `t` (i.e. `- 1e-4 == 0.0`), then the border_rotation will yield `0.0` hence no rotation.

i thought i remembered something in std::f32::consts::* for something about this but i cant find anything. It should prob be replaced by a more reliable constant.

its worth noting that it will still work even if we do something like `0.1` (`t - 0.1`) but the rotations are more precise if we decrease it

Reply:
`std::f32::consts::EPSILON` I think?

Reply:
ahhh yes lol, although now we have to do this:

https://github.com/voxell-tech/bevy_vello_graphics/pull/3/commits/861dd7f657aaabe0aea7a6f39a464ef6cef48ca3#diff-2773de0434a8335533e063c895e504359348ba765d78556a77cfed7e07d81a3bR157-R158

cause f64 is too small haha.